Add new operator gdrive to local (#14191)
mistercrunch/Airflow: docs/apache-airflow-providers-google/operators/transfer/gdrive_to_local.rst

    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.

Downloads data from Google Drive Storage to Local Filesystem
============================================================
The `Google Drive <https://www.google.com/drive/>`__ is
used to store daily use data, including documents and photos. Google Drive has built-in mechanisms to facilitate group work e.g.
document editor, file sharing mechanisms.

.. contents::
  :depth: 1
  :local:

Prerequisite Tasks
^^^^^^^^^^^^^^^^^^

.. include::/operators/_partials/prerequisite_tasks.rst

.. _howto/operator:GoogleDriveToLocalOperator:

GCSToLocalFilesystemOperator
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

:class:`~airflow.providers.google.cloud.transfers.gdrive_to_local.GoogleDriveToLocalOperator` allows you to download
data from Google Drive to local filesystem.

Below is an example of using this operator to download file from Google Drive to Local Filesystem.

.. exampleinclude:: /../../airflow/providers/google/cloud/example_dags/example_gdrive_to_local.py
    :language: python

>     :dedent: 0

Reviewer: ```suggestion
    :dedent: 4
```

Author: What does it mean? dedent

Reviewer: It will remove indentation from the example (remove leading spaces). So instead of this:
```py
    op = MyOperator(...)
```
documentation will include
```py
op = MyOperator(...)
```

>     :start-after: [START howto_operator_gdrive_download_file_task]
>     :end-before: [END howto_operator_gdrive_download_file_task]

Reviewer: ```suggestion
    :start-after: [START download_from_gdrive_to_local]
    :end-before: [END download_from_gdrive_to_local]
```
This has to be aligned with what we have in the example DAG.

Author: Yes, I'm already fixed this and previous
